test/search.cpp

```diff
@@ -63,6 +63,22 @@ class TempZimArchive : zim::unittests::TempFile
     }
 };
 
+std::string getSnippet(const zim::Archive archive, std::string query, int range) {
+    zim::Search search(archive);
+    search.set_suggestion_mode(false);
+    search.set_query(query);
+    search.set_range(0, range);
```

Comment:
As we always get the first result, it is better to set the range [0, 1]
(But it is small optimization)